tests/assemblers/test_svm.py

```diff
@@ -117,11 +117,23 @@ def kernel_ast(sup_vec_value):
                     ast.NumVal(sup_vec_value),
                     ast.FeatureRef(0),
                     ast.BinNumOpType.MUL),
-                ast.SqrtExpr(
-                    ast.BinNumExpr(
-                        ast.FeatureRef(0),
-                        ast.FeatureRef(0),
-                        ast.BinNumOpType.MUL)),
+                ast.IfExpr(
+                    ast.CompExpr(
+                        ast.SqrtExpr(
+                            ast.BinNumExpr(
+                                ast.FeatureRef(0),
+                                ast.FeatureRef(0),
+                                ast.BinNumOpType.MUL),
+                            to_reuse=True),
+                        ast.NumVal(0.0),
+                        ast.CompOpType.EQ),
+                    ast.NumVal(1.0),
+                    ast.SqrtExpr(
```

Comment:
[Nitpick] This part can be extracted into a separate variable and cached this way making expected AST a little simpler:
```
feature_norm = ast.SqrtExpr(
                            ast.BinNumExpr(
                                ast.FeatureRef(0),
                                ast.FeatureRef(0),
                                ast.BinNumOpType.MUL),
                            to_reuse=True)

...
ast.IfExpr(
                    ast.CompExpr(
                        feature_norm,
                        ast.NumVal(0.0),
                        ast.CompOpType.EQ),
                    ast.NumVal(1.0),
                    feature_norm)
```

Reply:
Sure, will do this in a separate PR.